spec/controllers/api/event_hours_controller_spec.rb

```diff
@@ -74,11 +75,13 @@ def expected_response
         'date' => date,
         'event_hours' => [{
           'event_hour_id' => event_hour.event_hour_id,
+          'capacity' => 5,
```

Comment:
Suggestion: Consider adding reserved value to expected_event_slots, e.g., through factory and show open_slots expected results as capacity - reserved.